packages/vue/jest.config.cjs

```diff
@@ -1,4 +1,19 @@
 module.exports = {
+  collectCoverage: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.(ts|vue)'],
+  // ignore coverage for index, version and style files
+  coveragePathIgnorePatterns: [
+    '<rootDir>/src/(components|composables|types)/index.ts',
```

Comment:
Shouldn't TS types not impact coverage by default?

Reply:
You are right but the type file is not pure in vue

Reply:
Got it, thanks for researching